packages/sodium/src/js/modules/TVerTypeHandler.js

```diff
@@ -4,33 +4,29 @@ export default class TVerTypeHandler {
         const duration = videojs.getPlayers()[Object.keys(videojs.getPlayers())[0]]
             .duration();
 
-        return !duration || Number.isNaN(duration) ? -1 : duration;
+        return !duration || Number.isNaN(duration) || !Number.isFinite(duration) ? -1 : duration;
```

Comment:
Number.isFinite(NaN)は偽なのでisNaNは不要ではないですか? 下記で良いかもと感じました:
```suggestion
        return duration && Number.isFinite(duration) ? duration : -1;
```

```suggestion
        return duration > 0 ? duration : -1;
```

Reply:
そうですね。更新しました。